src/views/TrialDescriptionPage/TrialDescriptionPage.jsx

```diff
@@ -28,7 +28,7 @@ import {
 	runQueryFetchers,
 } from '../../utilities';
 import ErrorPage from '../ErrorPage';
-import PageNotFound from '../PageNotFound/PageNotFound';
+import PageNotFound from '../ErrorBoundary/PageNotFound.jsx';
```

Comment:
TrialDescriptionPage is not handling ErrorsOccuredPage - when 500 response is mocked it renders PNF